latest_builds/matrix.yml

```diff
@@ -100,6 +117,20 @@ ansible-11:
       dists:
         - noble
         - oracular
+ansible-12:
+  packages:
+    - name: ansible-core
+      version_specifier: "~=2.19.0"
+      dists:
+        - noble
+        - oracular
+        - plucky
+    - name: ansible
+      version_specifier: "~=12.0"
+      dists:
+        - noble
+        - oracular
+        - plucky
```

Comment:
Does it work to already add 12, even though there are only pre-releases yet? (I have no idea :) )

Reply:
see [packaging specifiers](https://packaging.pypa.io/en/stable/specifiers.html) for more details if you're interested